crates/bin/pd/src/main.rs

```diff
@@ -129,15 +132,26 @@ async fn main() -> anyhow::Result<()> {
             );
 
             let tm_proxy = penumbra_tendermint_proxy::TendermintProxy::new(cometbft_addr);
-            let grpc_server = penumbra_app::rpc::router(&storage, tm_proxy, enable_expensive_rpc)?;
+
+            let grpc_routes = penumbra_app::rpc::routes(&storage, tm_proxy, enable_expensive_rpc)?
+                .into_axum_router()
+                .layer(
+                    ServiceBuilder::new().layer(TraceLayer::new_for_grpc().make_span_with(
```

Comment:
This addresses the fact that we can no longer make use of `tonic::transport::Server` and it's `trace_fn` setter (since we can no longer turn the "battries included server" back into an axum router).